xref-token/Cargo.toml

```diff
@@ -14,4 +14,5 @@ near-contract-standards = "3.1.0"
 
 [dev-dependencies]
 near-sdk-sim = "3.1.0"
-test-token = { path = "../test-token" }
\ No newline at end of file
+test-token = { path = "../test-token" }
+ref-token  = { path = "../ref-token" }
```

Comment:
We have test-token to help sim-test real token contract, so try to avoid using other token contract to test target token contract, as you can never tell the other token contract is OK or not, leave the test an unnecessary dependency.